Sources/MapboxNavigation/InstructionPresenter.swift

```diff
@@ -168,46 +171,65 @@ class InstructionPresenter {
         return attributedTextRepresentations.joined(separator: separator)
     }
     
-    func attributedString(forShieldComponent shield: VisualInstruction.Component.ImageRepresentation,
-                          repository: ImageRepository,
+    func attributedString(for representation: VisualInstruction.Component.ImageRepresentation,
+                          in repository: SpriteRepository,
                           dataSource: DataSource,
-                          cacheKey: String,
-                          onImageDownload: @escaping ImageDownloadCompletion) -> NSAttributedString? {
-        if let cachedImage = repository.cachedImageForKey(cacheKey) {
-            return attributedString(withFont: dataSource.font, shieldImage: cachedImage)
+                          onImageDownload: @escaping CompletionHandler) -> NSAttributedString? {
+        if let shield = representation.shield {
+            if shield.name == "us-state",
```

Comment:
The screenshots in https://github.com/mapbox/mapbox-navigation-ios/pull/3864#issue-1226127934 depict a generic shield for a California state route. Is this fallback working, or are the screenshots outdated perhaps?

Reply:
The `12A` is the exit view, which is generated by our SDK. And the `85` is the road using the Mapbox designed shield, so we can see it changes styles when Day-Nght style switch, and it matches the map rendering road shield as well.

Reply:
Following up in #3868.

Reply:
I see. The issue is #3868 and will be fixed by #3870